### PR TITLE
ROS1 in Ravestate

### LINKED ISSUE #116: ravestate_ros1
Just like `ravestate_ros2`, we need a `ravestate_ros` module which provides `RosPubProperty` and `RosSubProperty`.

For good measure, we should add `rospkg` to our Docker containers.

### PR DESCRIPTION
**Pull request by Toseban**
Fixes #116
The same interface between properties and ROS we have in ravestate_ros2, only this time with ROS1 :older_man: 
What's left to do:
- [x] Documentation in a Readme with examples :newspaper_roll: 
- [x] Tests of course, but maybe this is for another PR. Maybe we can do tests that run inside our beloved docker container that has ROS and ROS2? :package:

### CODE REVIEW

**Comment by josephbirkner** [comment]
Hmm, the test situation is hard to decide. Technically what we could do is use our docker image in travis. Then we could write amazing tests... Let's create an issue for that.

File: modules/ravestate_ros1/README.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
+```
```
Comment (josephbirkner) [inline modules/ravestate_ros1/README.md]: 🌈

**Comment by josephbirkner** [comment]
Follow-up: #120

**Comment by Toseban** [comment]
rospy is already in the docker image

**Pull request merged**